tsdfileapi/api.py

```diff
@@ -792,6 +824,9 @@ def handle_data(self, filemode: str, tenant: str) -> None:
             assert self.write_files(filemode, tenant)
             self.set_status(201)
             self.write({'message': 'data uploaded'})
+        except StorageTemporarilyUnavailableError:
+            self.set_status(503, reason="project_storage_migrating")
```

Comment:
For specifically the so-called "reason phrase" (`reason` here) in the HTTP status line, the convention is capitalized words separated by space, e.g. `404 Not Found` -- I'd go for `Project Storage Migrating` instead of the `project_storage_transitioning`.

Anyway, I also read from [the HTTP 1.1 spec.](//www.rfc-editor.org/rfc/rfc7230#page-22):

> A client SHOULD ignore the reason-phrase content

Which is kind of bad news if we want to rely on it as specification of the 503. That doesn't make the custom reason here useless entirely, just a bit discouraging we'd be going against the recommendation if parsing it.

Reply:
I'll change the phase to `Project Storage Migrating`, and add a custom header `X-Project-Storage: Migrating`, and then that has to be good enough, the deadline is fast approaching...

Reply:
Resolved in https://github.com/unioslo/tsd-file-api/pull/99/commits/acfce42b2bfec4a69a0c215005ef6354fe9f63f9